src/init-ceph.in

```diff
@@ -346,10 +346,10 @@ for name in $what; do
 
```

Comment:
Why not always 
<pre>
mkdir -p $fs_path
</pre>
even before testing the fs_type ? mkdir -p will succeed if the directory already exists, there is no need to check for its existence.